_includes/schedule.html

```diff
@@ -0,0 +1,83 @@
+<h2>Loveland Startup Day</h2>
+<h3>Sunday, May 31</h3>
+<div class="startup-day">
+  {% for event in site.data.schedule.events %}
+    <div class="startup-day-event">
+      <h4>
+        {{ event.title }}
+      </h4>
+      <div class="uk-grid">
+        <div class="uk-width-medium-1-4">
+          {% if event.day %}
+            <div class="day">
+              {{ event.day }}<br>
+              <span>{{ event.time|date:"%l:%M %p" }}</span>
+            </div>
+          {% endif %}
+          <div class="location">
+            {{ event.location }}<br>
+            <span><a href="//www.google.com/maps/place/{{ event.address|split:" "|join:"+" }},+Loveland,+CO+80537" target="_blank">
```

Comment:
If this gets `event.address` from `schedule.yml` above, it may yield strange results in Google map searches when the relevant address field in `schedule.yml` contains text not identifiable as an address by Google.